docs/requirements/firewall.md

```diff
@@ -24,4 +24,7 @@ For customers that are provisioning devices behind a firewall, there are a few a
 -   5228 
 -   5229 
 -   5230
-    
\ No newline at end of file
+
+-   3478, 5349 [TCP] (For TURN Server)
```

Comment:
Instead of this way mention something like:

URL: https://turn.shoonyacloud.com
Port:  3478, 5349 (TCP) and 3478, 5349, 49152 - 65535 (UDP)